routers/repo/download.go

```diff
@@ -59,6 +60,9 @@ func ServeData(ctx *context.Context, name string, size int64, reader io.Reader)
 	} else {
 		ctx.Resp.Header().Set("Content-Disposition", fmt.Sprintf(`attachment; filename="%s"`, name))
 		ctx.Resp.Header().Set("Access-Control-Expose-Headers", "Content-Disposition")
+		if filepath.Ext(name) == ".apk" && base.IsZipFile(buf) {
```

Comment:
Will not work with mixed case letters.

Reply:
I've fixed this with EqualFold.
Should I squash the fix and the original commit?